components/tidb_query_vec_expr/src/impl_string.rs

```diff
@@ -31,6 +31,24 @@ pub fn length(arg: &Option<Bytes>) -> Result<Option<i64>> {
     Ok(arg.as_ref().map(|bytes| bytes.len() as i64))
 }
 
+#[rpn_fn]
+#[inline]
+pub fn unhex(arg: &Option<Bytes>) -> Result<Option<Bytes>> {
+    if let Some(content) = arg {
+        // hex::decode will fail on odd-length content
+        // but mysql won't
+        // so do some padding
+        let mut padded_content = Vec::with_capacity(content.len() / 2 + content.len() % 2);
```

Comment:
Sorry I don't get it. Why the capacity is `content.len() / 2 + content.len() % 2`? I think it's at least the length of `content`...

Reply:
> Sorry I don't get it. Why the capacity is `content.len() / 2 + content.len() % 2`? I think it's at least the length of `content`...

Sorry, I mistaken it with the decoded result, fix very soon